altiProfil/www/css/altiProfil.css

```diff
@@ -1,14 +1,14 @@
 #mapmenu li.altiProfil #button-altiProfil span.icon{
-  background-image: url(getfile?targetmodule=altiProfil&file=img/noun_altitude.png);
+  background-image: url(img/noun_altitude.png);
```

Comment:
I don't see how it can work, as the url of altiProfil.css is certainly `/index.php/jelix/www/getfile?targetmodule=altiProfil&file=css/altiProfil.css` (or  `/something/index.php/jelix/www/getfile?targetmodule=altiProfil&file=css/altiProfil.css` if lizmap is installed into a subdirectory of the web site, like on lizmap.com)

Reply:
I haven't tried, and I'm not a module developer :)
I only took the behavior from https://github.com/3liz/lizmap-cadastre-module/commit/529af9c683ebc525d9cb9e505e16f006bf3ba118

Reply:
Ok, in fact:

into lizmap 3.5, the url is  `/something/index.php/jelix/www/getfile?targetmodule=altiProfil&file=css/altiProfil.css`, and into lizmap 3.6, the url is  `/something/index.php/jelix/res/altiProfil/css/altiProfil.css`.

So the url of the background image is  `/something/index.php/jelix/www/getfile?targetmodule=altiProfil&file=img/noun_altitude.png` or `/something/index.php/jelix/res/altiProfil/img/noun_altitude.png`.

So we should have 
`background-image: url(getfile?targetmodule=altiProfil&file=img/noun_altitude.png);`
or
`background-image: url(../img/noun_altitude.png);`

To avoid to drop support of lizmap 3.5 just for this url, probably the best solution is to copy assets into the www directory of lizmap during the installation and the upgrade.

Reply:
Or another solution : create a controller into altiprofil that mimics www.classic.php (by inherithing)...

Reply:
I have a patch